src/stamps/base.ts

```diff
@@ -36,6 +42,19 @@ export class BaseStampManager {
     throw new Error(ERROR_NO_STAMP)
   }
 
+  /**
+   * Start the manager in either hardcoded or autobuy mode
+   */
+  public async startFeature(
```

Comment:
Ehm, why this? 😳 Shouldn't this be part of `start` instead?!

Reply:
This was a nasty test left behind... sorry for this 👎